packages/hooks/feed/useFeedPrettyBalanceFee.ts

```diff
@@ -0,0 +1,36 @@
+import { useFeedPostFee } from "./useFeedPostFee";
+import { PostCategory } from "../../components/socialFeed/NewsFeed/NewsFeed.type";
+import { getStakingCurrency, parseUserId } from "../../networks";
+import { prettyPrice } from "../../utils/coins";
+import { useBalances } from "../useBalances";
+import { useSelectedNetworkId } from "../useSelectedNetwork";
+
+interface UseFeedPrettyBalanceFeeParams {
+  userId?: string;
+  category: PostCategory;
+}
+
+export const useFeedPrettyBalanceFee = ({
+  userId,
```

Comment:
this will most likely behave wrongly when there is no wallet connected, either split the balance and publishing fee in two hooks or take an additional networkId param

Reply:
Done